src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -5,12 +5,23 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace System.Net.Http.WinHttpHandlerTests
 {
+    public class testHandler:WinHttpHandler
```

Comment:
@josguil Coding standards require all class names to start with a capital letter.